app-editors/vis/vis-0.3.ebuild

```diff
@@ -18,6 +22,14 @@ DEPEND=" dev-libs/libtermkey
 	tre? ( dev-libs/tre:= )"
 RDEPEND="${DEPEND}"
 
+src_prepare() {
+	if use test; then
+		rm -r "${S}/test"
```

Comment:
As I said, all those lines need error handling (`||die`). Also why are you using an absolute path when you're already inside `${S}`?